core/src/services/rowService.js

```diff
@@ -11,14 +11,59 @@ module.exports = (db) => {
       return result;
     },
 
```

Comment:
@AbegaM - does this next block handle a single key being provided?

Reply:
@IanMayo are you referring to the `getById` function?

Reply:
Yes.

Reply:
yes it is working if you run this URL it returns the data

```
 localhost:8000/api/tables/albums/rows/1
```